selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/CheckWsAgentAfterStopProcessTest.java

```diff
@@ -38,18 +43,21 @@
  * @author Musienko Maxim
  */
 public class CheckWsAgentAfterStopProcessTest {
```

Comment:
I think we should rename the name of class because it a bit confuse. May be like: CheckResroingWorkspaceAfterStoppingWsAgentProcess or something like that. WDYF-?